File: objc_ex/objc_ex/ViewController.m
//
//  ViewController.m
//  objc_ex
//
//  Created by estgames on 2018. 5. 9..
//  Copyright © 2018년 estgames. All rights reserved.
//

#import "ViewController.h"
#import "objc_ex-Swift.h"

@interface ViewController ()

@end

@implementation ViewController
    EstTest * est;
- (void)viewDidLoad {
    [super viewDidLoad];
    est = [[EstTest alloc] initWithView:self];
    // Do any additional setup after loading the view, typically from a nib.
    //EstgamesCommon *test = [[EstgamesCommon] init];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)bannerShow:(id)sender {
    [est banerShow];
}
    
@end

File: testapp/testapp/ViewController.swift
//
//  ViewController.swift
//  testapp
//
//  Created by estgames on 2018. 1. 10..
//  Copyright © 2018년 estgames. All rights reserved.
//

import UIKit
import banner

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let test = bannerFramework()
        test.bannerAdd(pview: self.view)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

